synapse/handlers/room.py

```diff
@@ -37,6 +37,8 @@
 
 logger = logging.getLogger(__name__)
 
+id_server_scheme = "https://"
```

Comment:
When I am testing locally manually, changing this to http:// is much less hassle than me having to actually serve my sydent over SSL.

Not sure it's worth a comment...